### PR TITLE
Add variables to absinthe spans

### PR DESCRIPTION
**Pull request by antonsatin**
This adds ability to see variables that were passed along with query in
absinthe spans. This functionality must be explicitly enabled in the
config.

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
**Once this PR has been reviewed and has the lgtm label**, please ask for approval from [antonsatin](https://github.com/antonsatin). For more information see [the Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: test/mv_opentelemetry/absinthe_test.exs [hunk 1]
```diff
@@ -38,6 +40,7 @@ defmodule MvOpentelemetry.AbsintheTest do
     assert {"graphql.field.name", "human"} in attributes
     assert {"service.component", "test.harness"} in attributes
     assert {"graphql.field.schema", MvOpentelemetryHarnessWeb.Schema} in attributes
+    assert {"graphql.operation.variables", Jason.encode!(variables)}
```
Comment (maciej-szlosarczyk) [inline test/mv_opentelemetry/absinthe_test.exs:43]: This assertion is wrong, you're checking the if the tuple is 'truthy' here.

File: lib/mv_opentelemetry/absinthe.ex [hunk 2]
```diff
@@ -83,8 +83,19 @@ defmodule MvOpentelemetry.Absinthe do
 
   def handle_event([:absinthe, :execute, :operation, :start], _measurements, meta, opts) do
     event_name = Enum.join([opts[:prefix]] ++ [:execute, :operation], ".")
+    variables = meta |> Access.get(:options, []) |> Keyword.get(:variables, %{})
 
-    attributes = [{"graphql.operation.input", meta.blueprint.input}] ++ opts[:default_attributes]
+    attributes =
+      [
+        {"graphql.operation.input", meta.blueprint.input}
```
Comment (maciej-szlosarczyk) [inline lib/mv_opentelemetry/absinthe.ex:90]: Something happened with this indentation here 😿

**Review by maciej-szlosarczyk — DISMISSED** [review]

**Comment by sauron-droid** [comment]
New changes are detected. LGTM label has been removed.

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
**Once this PR has been reviewed and has the lgtm label**, please ask for approval from [antonsatin](https://github.com/antonsatin). For more information see [the Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
**Once this PR has been reviewed and has the lgtm label**, please ask for approval from [antonsatin](https://github.com/antonsatin). For more information see [the Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by maciej-szlosarczyk — APPROVED** [review]

**Comment by sauron-droid** [comment]
LGTM label has been added.  <details>Git tree hash: e8c2e29f9cbbd9009d01bc65c31e2aac5bd1a26e</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/mv-opentelemetry/pull/61#pullrequestreview-1739544827" title="Approved">maciej-szlosarczyk</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)~~ [maciej-szlosarczyk]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**